libvast/test/time.cpp

```diff
@@ -76,8 +76,12 @@ TEST(fractional durations) {
   check_timespan("-42.001ms", -42001us);
 }
 
-TEST_DISABLED(compound durations) {
+TEST(compound durations) {
   check_timespan("3m42s10ms", 3min + 42s + 10ms);
+  check_timespan("-10m8ms1ns", -10min + 8ms + 1ns);
```

Comment:
```suggestion
  check_timespan("3s42s10ms", 3s + 42s + 10ms);
  check_timespan("42s3m10ms", 3min + 42s + 10ms);
  check_timespan("-10m8ms1ns", -10min + 8ms + 1ns);
```
These additional test pass as is. Do we wan't the parser to be relaxed?